tests/consumers/test_message_processors.py

```diff
@@ -71,3 +74,54 @@ def test_message_processors(
             for line in rust_processed_message.rstrip(b"\n").split(b"\n")
             if line
         ] == python_processed_message.rows
+
+
+def test_replay_processor() -> None:
```

Comment:
I think instead of adding this additional test, we should add a new parametrization to the above test

Reply:
It had enough of its own peculiarities that I added a different test.  I think this is better than adding more if statements to the original test.

Reply:
how about this: we could add a third parameter (next to topic and processor) that defines a normalization function, and as a result get rid of the if-statements in the main test as well:

```
@pytest.mark.parametrize("topic,processor,normalize_row", [...])
```

Reply:
I took @phacops advice and updated the python consumer to return a more sensible type (datetime -> u32 timestamp).  This let me drop the normalize function and re-use the test case.